### PR TITLE
WT-4568 Fix operation tracking visualization.

### PR DESCRIPTION
**Pull request by fedorova**
Used to display callstacks incorrectly.

### CODE REVIEW

File: tools/optrack/find-latency-spikes.py [hunk 1]
```diff
@@ -280,13 +282,13 @@ def assignStackDepths(dataframe):
 
     for i in range(len(df.index)):
 
-        myStartTime = df.at[i, 'start'];
+        myEndTime = df.at[i, 'end'];
 
         # Pop all items off stack whose end time is earlier than my
```
Comment (luke-pearson) [inline tools/optrack/find-latency-spikes.py]: Comment here needs to be updated for end time

Comment (luke-pearson) [reply tools/optrack/find-latency-spikes.py]: Other than that this looks good to me

**Comment by fedorova** [comment]
@luke-pearson, thank you for reviewing the branch. I pushed the comment update that you suggested.  Autobuild fails for mysterious reasons. The python build shows as failed, but if you zoom in, there are no problems identified. How do you suggest to proceed? May I merge the branch? Should I wait? Thanks!

**Comment by fedorova** [comment]
@luke-pearson, forgot to mention, the evergreen build says that the patch must be manually authorized. Not sure what the protocol here is. Can I do this? What are the rules?

**Comment by keithbostic** [comment]
autobuild: retest this please

**Comment by lukech** [comment]
@fedorova you might have some evergreen credential issue there (if failing to trigger an evergreen build). Can you send me an email with screenshots of your problem?

**Pull request merged**